api/v1beta1/hetznercluster_webhook.go

```diff
@@ -20,12 +20,13 @@ import (
 	"fmt"
 	"reflect"
 
-	"github.com/syself/cluster-api-provider-hetzner/pkg/utils"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	"github.com/syself/cluster-api-provider-hetzner/pkg/utils"
 )
```

Comment:
@prometherion just out of curiosity, why did you re-order the imports? Which tool do you use for this? If it makes sense, we could integrate this in our IDE/CI, too.

Reply:
I'm using `goimports` locally with the `-l` flag.

It seems already enabled in the `golangci-lint` file, I'll configure it properly and open a different PR to keep this atomic to the issue mentioned.

Reply:
#849